data/human/fleets.txt

```diff
@@ -3488,3 +3488,105 @@ fleet "Human Wormhole Guards"
 		"Frigate (Mark II)"
 	variant 5
 		"Rainmaker (Mark II Old Weapons)"
+
+fleet "Derelict Republic"
```

Comment:
I think it makes more sense to use the same regional specifiers that other fleets use. That is, have the fleets be named Derelict Northern/Core/Southern rather than Republic/Syndicate/Southern. Same with the outfitters.